### PR TITLE
Link measurements with samples and users

### PR DESCRIPTION
**Pull request by maasencioh**
### Measurements list
image

### Measurement detail
image

### CODE REVIEW

File: addons/events/Schemas/Event.graphql [hunk 1]
```diff
@@ -5,6 +5,7 @@ type Query {
     filterBy: EventFilterInput
     sortBy: EventSortInput
   ): EventPage!
+  event(id: String!): Event!
```
Comment (targos) [inline addons/events/Schemas/Event.graphql]:
```suggestion
  event(id: ID!): Event!
```

https://graphql.org/learn/schema/#scalar-types

File: app/Schemas/Measurement.graphql [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  username: String!
+  sampleCode: [String!]!
+  createdBy: String!
+  fileId: String
+  description: String
+  createdAt: DateTime!
+}
+
+type GeneralMeasurement implements Measurement {
+  id: String!
```
Comment (targos) [inline app/Schemas/Measurement.graphql]:
```suggestion
  id: ID!
```

File: app/Schemas/Measurement.graphql [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+# Measurement specific data
+enum MeasurementTypes {
+  general
+  transfer
+}
+
+interface Measurement {
+  id: String!
```
Comment (targos) [inline app/Schemas/Measurement.graphql]:
```suggestion
  id: ID!
```

File: app/Schemas/Measurement.graphql [hunk 4]
```diff
@@ -0,0 +1,61 @@
+type Query {
+  measurements(
+    type: MeasurementTypes
+    limit: Int
+    skip: Int
+    filterBy: MeasurementFilterInput
+    sortBy: MeasurementSortInput
+  ): MeasurementPage!
+  measurement(id: String!): Measurement!
```
Comment (targos) [inline app/Schemas/Measurement.graphql]:
```suggestion
  measurement(id: ID!): Measurement!
```

**Review by targos — COMMENTED** [review]
I won't have much time to review everything. Please tell me if you want me to look at a specific part of the diff.

**Comment by maasencioh** [comment]
I'll merge this and work with more specific and small changes on other branches

**Pull request merged**